.github/workflows/publish.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish Python 🐍 distributions 📦 to PyPI
+on:
+  release:
+    types: [created]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get tagged version
+        id: vars
+        run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
+      - name: Replace tag template in `setup.py`
+        env:
+          RELEASE_VERSION: ${{ steps.vars.outputs.tag }}
+        run: |
+          echo replacing TEMPLATE_VERSION in setup.py with $RELEASE_VERSION...
+          sed -i "s|TEMPLATE_VERSION|$RELEASE_VERSION|" setup.py
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel twine
+      - name: Build and publish
+        env:
+          TWINE_USERNAME: __token__
+          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+        run: |
+          python setup.py sdist bdist_wheel
+          python -m twine upload dist/*
```

Comment:
Is each run of the workflow guaranteed to run in a fresh environment? No way they'll cache the 'dist' dir? I'd expect it to be pristine, but who knows :) Just curious if we should add '--skip-existing' or not. No big deal, just curious.

Reply:
Hm, i'm a bit unfamiliar with `--skip-existing`, haven't had to use it - are you asking if releases are guaranteed to be unique? Or am I misunderstanding this?